pkgs/by-name/ku/kubectl-validate/package.nix

```diff
@@ -24,6 +24,9 @@ buildGoModule {
   preBuild = ''
     mv cmd/download-builtin-schemas/main.go cmd/download-builtin-schemas/_main.go
     mv pkg/openapiclient/github_builtins_test.go pkg/openapiclient/_github_builtins_test.go
+
```

Comment:
Can you add a comment why this is not part of the build? 
If its not part of the tests, then I think thats not the way to do it.